share/kak/kakrc

```diff
@@ -25,6 +25,10 @@ def -params 1 -docstring "colorscheme <name>: enable named colorscheme" \
     fi
 }}
 
+map global normal <paste:begin> '\i'
+map global insert <paste:begin> '<a-;>\i'
+map global insert <paste:end>   <esc>
```

Comment:
Those should not be mappings, but built-ins in the C++ code, the user can override the mappings as we do for any other commands.